### ISSUE TITLE
Naming issue

### ISSUE THREAD

**Issue by pascaldevink** [SilexLab/Silex#13]
I just noticed your framework, congrats with the open source project!
But one can not help but notice that the name looks a lot like the name of that other framework project called Silex (http://silex.sensiolabs.org).

Was this done deliberately or did you just not know about it? In either case, don't you think another name would avoid confusion and give your framework some more room to grow on its own?

**Comment by noxifoxi** [comment]
Thank you for your information.
The matter is, we know for quite some time about this, but it was unintentional.
This has to do with the old project SilexBoard from which Silex was created.

I changed the project plan to create at first a framework called Silex and if the framework is ready for productive use we will continue the work on SilexBoard and other projects like SilexBlog on the basis of the framework (Silex).

Now we are searching for a new unique (Latin) name for our framework and upcoming products (<name>Board, <name>Blog, etc...).

tl;dr: It was unintentional, and we are brain-storming for a new name.
